build.sbt

```diff
@@ -587,7 +587,8 @@ lazy val elasticsearchPlugin = project
     addCompilerPlugin(betterMonadicFor),
     assembly / assemblyOption  := (assembly / assemblyOption).value.copy(includeScala = false),
     assembly / test            := {},
-    addArtifact(Artifact("delta-elasticsearch-plugin", "plugin"), assembly)
+    addArtifact(Artifact("delta-elasticsearch-plugin", "plugin"), assembly),
+    run / fork                 := true
```

Comment:
why the fork on the run?

Reply:
Because of getting weird errors with Slick: https://github.com/slick/slick/issues/1578

Reply:
But you are right, it should be on tests, not on run :)